RecoTracker/LSTCore/standalone/setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+###########################################################################################################
+# Setup environments
+###########################################################################################################
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $DIR/code/rooutil/thisrooutil.sh
+
+ARCH=$(uname -m)
+if [[ $(hostname) == *lnx4555* ]]; then
+  export SCRAM_ARCH=el9_amd64_gcc12
+elif [[ $ARCH == "aarch64" || $ARCH == "arm64" ]]; then
+  export SCRAM_ARCH=el9_aarch64_gcc12
+else
+  export SCRAM_ARCH=el8_amd64_gcc12
+fi
+export CMSSW_VERSION=CMSSW_14_1_0_pre3
```

Comment:
Is this ok if we leave it like that? Maybe we should just grab the version of CMSSW we have set up?

Reply:
We should have a better setup script. I only made the minimal modifications to get it to work.